index.bs

```diff
@@ -4514,6 +4541,7 @@ failed</dfn> steps given |context| and |navigation status|:
         browsingContext.UserPromptClosedParameters = {
           context: browsingContext.BrowsingContext,
           accepted: bool,
+          type: "alert" / "confirm" / "prompt" / "beforeunload",
```

Comment:
nit: perhaps it makes sense to extract this into a separate type and re-use between UserPromptClosedParameters and UserPromptOpenedParameters to make sure they are always the same?